hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestContainerEndpoint.java

```diff
@@ -229,13 +239,63 @@ public void setUp() throws Exception {
     OMMetadataManager omMetadataManagerMock = mock(OMMetadataManager.class);
     Table tableMock = mock(Table.class);
     when(tableMock.getName()).thenReturn("KeyTable");
-    when(omMetadataManagerMock.getKeyTable(getBucketLayout()))
+    when(omMetadataManagerMock.getKeyTable(BucketLayout.DEFAULT))
```

Comment:
same applies here as above

Reply:
Thanks for the comment @dombizita since both Legacy and Default serve the same purpose, I might have jumbled them up in places. I have made the required changes do take a look !!